experiments/expXXX_create_speed_run_caches.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
What's the XXX supposed to be in the name?

Reply:
renamed. was gonna make an experiment but decided to give it a name more like pretraining_datasets